cli.py

```diff
@@ -7,11 +7,11 @@
 from giskard_cicd.loaders import GithubLoader, HuggingFaceLoader
 from giskard_cicd.pipeline.runner import PipelineRunner
 
-from automation import create_discussion
+from automation import create_discussion, init_commit_scheduler, commit_to_dataset
 
 
 logger = logging.getLogger(__file__)
-
+scheduler = init_commit_scheduler()
```

Comment:
In case of not login (`ValueError` or some other exceptions), does the program stop?

Reply:
I have updated the code here to avoid stop the process at all if token or leaderboard dataset missing.